test/openAPI/features/g2p_bulk_payment.feature

```diff
@@ -0,0 +1,67 @@
+@method=POST @endpoint=/bulk-payment
+Feature: Handover a batch of credit instructions to be processed
+
+This API is to be exposed by the Payments BB.
+It will be called by the Source BB to handover a batch of credit instructions to be processed.
+
+    @smoke
+    Scenario: Succesfully handovered a batch of credit instructions to be processed smoke type test
+        Given Source BB wants to handover a batch of credit instructions to be processed
+        When POST request to bulk-payments with given "stringstring" as RequestID, "stringstring" as SourceBBID, "stringstring" as BatchID, "stringstringstri" as InstructionID, "stringstringstringst" as PayeeFunctionalID, 100 as Amount and "USD" as Currency is sent
```

Comment:
Is it possible to change every "stringstring...." to something more specific, e.q. `"requestID123" as RequestID`?
In the testing web app, this will not be user friendly.